features/buyers/applications/new.feature

```diff
@@ -0,0 +1,167 @@
+@javascript @ignore-backend
+Feature: Create application
+  In order to control the way my buyers are using my API
+  As a provider
+  I want to create their applications
+
+  Background:
+    Given a provider "foo.3scale.localhost"
+    And provider "foo.3scale.localhost" uses backend v2 in his default service
+    And provider "foo.3scale.localhost" has multiple applications enabled
+    And provider "foo.3scale.localhost" has "service_plans" switch allowed
```

Comment:
Basically knowing the domain `foo.3scale.localhost` is not really helpful in any case. 
Following comment applies to the whole file not only to `foo.3scale.localhost`


The reader needs to know what and not the implementation details. We try to remove useless information from cucumber and focus on readable business logic https://cucumber.io/docs/bdd/better-gherkin/

Cucumber should be readable and get to the point of the feature tested, hiding all small steps is the task of the developer not the reader.

A good example is from this file: https://github.com/3scale/system/pull/6977/files#diff-866bfaaad5caa1bcbeda34bcd98bbf6e80457aa4d26c47dedbe0f09811a56ade

To this file https://github.com/3scale/system/pull/6977/files#diff-1900a5761627b919a04a7650ffbb6c57ce4aa0051941bdbb3054dedd8e928acd

But I understand it is much work for already existing cucumbers. You can do them on another PR. For new cucumbers however please do the new correct way.

Reply:
OK in that case, I am going to restore it and leave inside `old`. That way at least we know the Feature needs to be properly refactored.